nimbus_nlp/question_classifier.py

```diff
@@ -12,97 +12,6 @@
 class QuestionClassifier:
     def __init__(self, file_path="question_set_clean.csv"):
         self.classifier = None
```

Comment:
Did you delete all this stuff from question_classifier?

Reply:
Yes. The last PR re added the old feature extraction functions and I removed those. The other functions are just moved within the class. You can check the full text of the file to see no functionality had been removed.